src/configure

```diff
@@ -184,6 +189,29 @@ do
   --cudatk-dir=*)
     CUDATKDIR=`read_dirname $1`;
     shift ;; #CUDA is used in src/cudamatrix and src/nnet{,bin} only
+  --host=*)
+    # We are going to prepend this to compilation tool invocations, so we need to take
+    # the specified host triple and at a '-'
+    HOST=`expr "X$1" : '[^=]*=\(.*\)'`-;
+    shift ;;
+  --fst-version=*)
+    OPENFST_VER=`expr "X$1" : '[^=]*=\(.*\)'`;
+    shift;;
+  --target-arch=*)
+    TARGET_ARCH=`expr "X$1" : '[^=]*=\(.*\)'`;
+    shift ;;
+  --android)
```

Comment:
I see why you need a flag like this but I don't like that we will have a special flag for android cross-compilation. I mean I think we should have a more general way of specifying that we are cross compiling (something like a "--target=*" flag) and then check what the target platform is and do the things necessary for that platform. What do you think?